azure-devops/run_build.yml

```diff
@@ -0,0 +1,51 @@
+parameters:
+  targetPlatform: 'x64'
+
+jobs:
+- job: vs2019_hosted_${{ parameters.targetPlatform }}
```

Comment:
vs2019_hosted_ is kind of a misnomer now; can we just use the target platform itself as the job title?